### PR TITLE
Add test coverage report

### PR DESCRIPTION
**Pull request by ivangergo3**
Added similar test-coverage report as we have in "cound-frontend"

So from now on, all tests will be executed when a pull request is being created.

And we will publish the results into google cloud, so we can check the current status [here](https://github.com/netdata/test-automation/wiki/Netdata-Test-Coverage-Report)

Could you please check this change? And if you have any concerns let me know

### CODE REVIEW

File: .github/workflows/tests.yml
Comment (novykh) [inline .github/workflows/tests.yml:5]: Do it only on push on master.

**Review by novykh — CHANGES_REQUESTED** [review]

**Review by burbuli8ra — APPROVED** [review]
LGTM, but please wait for 1 more approval before merging!
Also, I see that this branch is out of sync with the base! Maybe you could do a rebase to avoid conflicts! 😄

**Review by novykh — APPROVED** [review]

**Comment by ivangergo3** [comment]
@jjtsou do you think it can be merged now?

**Review by jjtsou — APPROVED** [review]

**Comment by ivangergo3** [comment]
@novykh  do you think it can be merged now?

**Comment by novykh** [comment]
Yes whatever is approved can be merged.

**Pull request merged**
